common/src/JetIds.cxx

```diff
@@ -682,60 +681,27 @@ bool JetPUid::operator()(const Jet & jet, const Event &ev) const{
 HotZoneVetoId::HotZoneVetoId(const bool& isHotZoneOnly) {
   std::string mapHistname;
 
-  for (const auto& el : year_str_map) {
-    std::string year = el.second;
-    if (year.find("UL")!=std::string::npos) {
-      mapHistname = isHotZoneOnly? "h2hot_ul17": "h2hot_ul17_plus_hep17";
-      try {
-        TFile file_map(locate_file("JECDatabase/hotzone_maps/Summer19UL17_V1/hotjets-UL17.root").c_str());
-        h2HotExcl[year] = (TH2D*) file_map.Get(mapHistname.c_str());
-        h2HotExcl.at(year)->SetDirectory(0);
-        file_map.Close();
-      } catch (...){}
-
-      continue;
-    }
+  std::string year = "UL16postVFP";
+
+  TFile file_map(locate_file("/nfs/dust/cms/user/amalara/WorkingArea/UHH2_106X_v1_UL/CMSSW_10_6_13/src/UHH2/DiJetJERC/conf/hotjets-UL16.root").c_str());
```

Comment:
Please make this filepath user-independent

Reply:
Thanks for pointing that out.
I thought for a bit if it made sense or not. I decided for not for the following reasons:
1) That file would eventually go into the JEC database. Right now it has only circulated "internally" for the JERC studies.
2) This module is not recommended for anyone in our group, but used for the JERC studies only. (Aka. Me and Alex at the moment)
3) I hope the recommendations on which file to use are going to be finalised in the next 2 weeks. Afterwards I will make sure this module is used properly.
4) Adding now and then remove a root file in a couple of weeks is not really nice for the git idea.

I had to modify the module otherwise the code was crushing, given the update. And it's also needed for me and Alex. I also though if I should comment the module at all. I hope I can fix it shortly.
 
Said that, I agree it's a bit ugly. Although I'm not leaving the group in the next two weeks.
What do you think? 

The other solution would be to leave this PR open for a couple of weeks. But then we should be careful of not making any merge conflict! I'm open for suggestions ;)

Reply:
Given your explanations, it's okay to just merge it now I think. But we should make sure to not forget updating that filepath in a few weeks once that hotzone map for UL16 is added in the JECDatabase (if I understand correctly). In order to make sure to not forget it, could you maybe add a std::cout or runtime warning to the module? Just an idea

Reply:
Hi @anmalara @MatthiesC do you want me to merge the PR like this or do you prefer to add a cout like suggested?